### ISSUE TITLE
goroutine 泄露

### ISSUE THREAD

**Issue by qydysky** [qydysky/bili_danmu#75]
```
32 @ 0x439056 0x468995 0x9c9c55 0x46ba61
#	0x468994	time.Sleep+0x134					/opt/hostedtoolcache/go/1.20.5/x64/src/runtime/time.go:195
#	0x9c9c54	github.com/qydysky/bili_danmu/Reply.ShowRevf+0x2b4	/home/runner/work/bili_danmu/bili_danmu/Reply/F.go:135
```

**Comment by qydysky** [comment]
fix 761a870505799a64c643bdb5a61fd031f9b688a7

**Comment by qydysky** [comment]
fix
19705632d69316cf7527e5c103ef0c368d724c2b
